pkg/lokifrontend/frontend/v1/frontend.go

```diff
@@ -118,6 +118,7 @@ func New(cfg Config, limits Limits, log log.Logger, registerer prometheus.Regist
 }
 
 func (f *Frontend) starting(ctx context.Context) error {
+	f.subservicesWatcher = services.NewFailureWatcher()
```

Comment:
How does this change in `Frontend` relate to the rest of the PR which is about TLS settings?

Reply:
I added this in order to pass the test cases.

This change had to be made as a result of update `weaveworks/common`. In the newer version (which contained the TLS properties), an error/panic to occur when invoking the function without a watcher.